pkg/crc/config/settings.go

```diff
@@ -145,14 +139,14 @@ func GetPreset(config Storage) preset.Preset {
 }
 
 func defaultNetworkMode() network.Mode {
-	if version.IsInstaller() {
+	if runtime.GOOS != "linux" {
 		return network.UserNetworkingMode
 	}
 	return network.SystemNetworkingMode
 }
 
 func GetNetworkMode(config Storage) network.Mode {
-	if version.IsInstaller() {
+	if runtime.GOOS != "linux" {
 		return network.UserNetworkingMode
```

Comment:
This hunk can be dropped together with the first one. If you change this, this means non-release builds will always be using usermode networking, even if there is a configuration option for it.